oracle/oci-utils
# LINUX-407 - LINUX-1615

This changes are about creating and deleting Libvirt virtual network based on OCI vNIC 
This introduce new commands
- create-network
- delete-network
Creates and delete Libvirt virtual network
exemple
/bin/oci-kvm create-network -n 10.0.0.64 --network-name manunet -F 10.0.1.100 -S 10.0.1.101 -E 10.0.1.200 -P 24 --vcn-cidr 10.0.0.0/16

This introduce a new option for guest creation : --virtual-network  
When used , guest is attached to the specified virtual network 
ex: 
/bin/oci-kvm create --disk-size 8 --pool sdb --virtual-network mynet -D foo2 --virt --vcpus 1 

test done : 
- create a network
- delete a network
- create two guests on virtual network and verify their network interfaces. Also check that they can ping each other.
